src/test/fuzz/netbase_dns_lookup.cpp

```diff
@@ -64,7 +64,7 @@ FUZZ_TARGET(netbase_dns_lookup)
     }
     {
         CSubNet resolved_subnet;
-        if (LookupSubNet(name, resolved_subnet, fuzzed_dns_lookup_function)) {
+        if (LookupSubNet(name, resolved_subnet)) {
```

Comment:
What is the reasoning behind this parameter no longer being needed?

Reply:
It is here: https://github.com/bitcoin/bitcoin/pull/23219#discussion_r763894498, `LookupSubNet()` never does lookups.